pkg/config/config_template.yaml

```diff
@@ -1068,6 +1068,19 @@ api_key:
   #   process: 10
   #   process_realtime: 2
 
+  ## @param process_discovery - custom object - optional
+  ## Specifies custom settings for the process_discovery object
+  # process_discovery
+      ## @param enabled - boolean - optional - default: enabled
+      ## Toggles the process_discovery check. This check is used to gather information about possible intergrations
+      ## that can be enabled by the user
+      # enabled: true
+
+      ## @param interval - duration - optional - default: 4h
+      ## An interval, in hours, that speficies how often the process discovery check should run
```

Comment:
```suggestion
      ## An interval in hours that specifies how often the process discovery check should run.
```

Reply:
speficies 😭

Thank you so much for fixing my bad grammar